src/main/java/com/nylas/Events.java

```diff
@@ -80,6 +80,70 @@ public Event rsvp(String eventId, String status, String accountId, String commen
 		return client.executePost(authUser, url, params, modelClass);
 	}
 
+
+	/**
+	 * Generate an ICS file server-side, from an Event
+	 * {@code icsOptions} is set to null.
+	 * @see Events#generateICS(String, ICSOptions)
+	 */
+	public String generateICS(String eventId) throws RequestFailedException, IOException {
+		return generateICS(eventId, null);
+	}
+
+	/**
+	 * Generate an ICS file server-side, from an Event ID
+	 * @param eventId The ID of the existing Event to generate an ICS file for
+	 * @param icsOptions Configuration for the ICS file
+	 * @return String for writing directly into an ICS file
+	 * @see <a href="https://developer.nylas.com/docs/api/#post/events/to-ics">Generate ICS File</a>
+	 */
+	public String generateICS(String eventId, ICSOptions icsOptions) throws RequestFailedException, IOException {
+		if(eventId == null) {
```

Comment:
do you have guava or other helper libraries? I think java 11 has isNullorEmpty, or isBlank to check for empty strings

Reply:
@jseller thanks! We're targeting Java 8 and I don't want to introduce Guava for just this, but I did forget that we have a `nullOrEmpty` method for strings in our `Validations` class. Using this now, thanks and good call!